Error: 
  â”Œ tests/js_error/source_locations/expression/yield_star/get_iterator_throws.js:3:11
  |
3 |     throw new Error();
  |           ^
Stack trace:
  at <anonymous> (tests/js_error/source_locations/expression/yield_star/get_iterator_throws.js:3:11)
  at generator (tests/js_error/source_locations/expression/yield_star/get_iterator_throws.js:8:3)
  at next (<native>)
  at <global> (tests/js_error/source_locations/expression/yield_star/get_iterator_throws.js:12:1)